### PR TITLE
fix: isolate-vm segfault errors by moving docker base image to ubuntu

### PR DESCRIPTION
**Pull request by psrikanth88**
## Description of the change
Build docker images over ubuntu base. Suspecting that alpine might be using different C++ runtime shared libraries like glibc vs muslc

CFD Link: https://www.notion.so/rudderstacks/hosted-multi-tenant-transformer-pods-are-in-crashloop-4e21cab081934749beccd33ce96cee3d

Discussion for context: https://rudderlabs.slack.com/archives/C02HCFVJVAR/p1660828860955409

Tested with the payload that was causing crashing on the current transformer version

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,18 @@
-FROM node:14.19.0-alpine3.15
+FROM ubuntu:20.04
 
-RUN apk update
-RUN apk upgrade
+RUN apt-get update \
+  && apt-get install -y curl make g++ \
+  && curl -sL https://deb.nodesource.com/setup_14.x | bash \
+  && apt-get install -y nodejs \
```
Comment (utsabc) [inline Dockerfile:6]: Which version of node we are installing here?

File: Dockerfile [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 
-# installing specific make version based on your previous configuration
-RUN apk add make=4.2.1-r2 --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main
-
-# installing specific gcc version based on your previous configuration
-RUN apk add g++=9.3.0-r0 --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main
-
-# RUN apk add --no-cache tini python make g++
-RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
+RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app && mkdir -p /home/node/.npm &&  chown -R node:node /home/node/.npm
```
Comment (utsabc) [inline Dockerfile:15]: Would helpful to understand context with some comment

File: Dockerfile (hunk 1 above)
Comment (saikumarrs) [reply Dockerfile:6]: https://deb.nodesource.com/setup_14.x

Comment (koladilip) [reply Dockerfile:6]: @Jayachand did we try with ubuntu 22.04?

Comment (koladilip) [reply Dockerfile:6]: As we are see memory increase with ubuntu 20.04, just a suggestion, should we try with 22.04 and see?

**Pull request closed without merging**